### DISCUSSION TITLE
Question on StitchedMeshGenerator

### DISCUSSION THREAD

**Discussion by TheGreatCid** [idaholab/moose#24832] [Q&A General] [answered]
Hi all,

When using StitchedMeshGenerator, it works great when I only have one boundary that needs to be combined between two meshes. However, in the problem I have, the two meshes have two boundaries that need to be stitched with each other. Is there a functionality for this?

**Comment by GiudGiud** [comment, marked as answer]
Hello

No you ll need to rename the two boundaries to one before stitiching, on both meshes, using a RenameBoundaryGenerator

Guillaume

**Reply by TheGreatCid** [reply]
To make sure I understand:
I have 4 total boundaries to consider on mesh A and mesh B. I want to stitch boundary ID's A1->B1 and A2->B2. Are you saying to rename the A2->B2 boundaries to be A1->B1? 

edit: My wording was unclear in my original question. So I am considering four total boundaries. Two on each mesh

**Reply by GiudGiud** [reply]
yes. Make sure the boundaries on mesh 1 have the same name, and on mesh 2 as well

**Reply by TheGreatCid** [reply]
That worked, thanks!
